dined/data_1D.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+"""
+This module is a client that allows the user to interact with the dined API
+This should be a swagger client in the future
+"""
+import json
+import requests
+from requests.exceptions import HTTPError
+from dined.config import BASE_URL
```

Comment:
I get this error when running the GettingDinedData notebook:

`---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/Users/tohu/Projects/GitHub/dyned/notebooks/GettingDinedData.ipynb Cell 5' in <cell line: [1]()>()
----> 1[ import dined.data_1D as data

File ~/Projects/GitHub/dyned/dined/data_1D.py:9, in <module>
      ]()[7]()[ import requests
      ]()[8]()[ from requests.exceptions import HTTPError
----> ]()[9]()[ from dined.config import BASE_URL
     ]()[10]()[ # import hashlib
     ]()[12]()[ def raw_issue_request(method, url, data=None, binary=False):

ModuleNotFoundError: No module named 'dined.config']()`